src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -141,13 +143,32 @@ def test_load_returns_expected_data(self):
         addon, block = self._blocked_addon()
 
         notblocked_version = addon.current_version
-        block_version = self._block_version(
-            block, self._version(addon), block_type=BlockType.BLOCKED
-        )
+
+        blocked_versions = [
+            self._block_version(
+                block, self._version(addon), block_type=BlockType.BLOCKED
+            )
+            for _ in range(10)
+        ]
+        soft_blocked_versions = [
+            self._block_version(
+                block, self._version(addon), block_type=BlockType.SOFT_BLOCKED
+            )
+            for _ in range(10)
+        ]
 
         mlbf_data = MLBFDataBaseLoader(self.storage)
         assert mlbf_data[MLBFDataType.BLOCKED] == MLBF.hash_filter_inputs(
-            [(block_version.block.guid, block_version.version.version)]
+            [
+                (block_version.block.guid, block_version.version.version)
+                for block_version in blocked_versions
+            ]
+        )
+        assert mlbf_data[MLBFDataType.SOFT_BLOCKED] == MLBF.hash_filter_inputs(
+            [
+                (soft_blocked_version.block.guid, soft_blocked_version.version.version)
+                for soft_blocked_version in soft_blocked_versions
+            ]
```

Comment:
can we assert the `len()` as well? it should 10 for each type, right?

Reply:
The len is asserted via the equality check but I cleaned up the code a bit to make it more obbvious by using a single 10 length loop